recipes/cubicinterpolation/all/conanfile.py

```diff
@@ -39,7 +39,7 @@ def configure(self):
             del self.options.fPIC
 
     def requirements(self):
-        self.requires("boost/1.75.0")
+        self.requires("boost/1.72.0")
```

Comment:
```suggestion
        self.requires("boost/1.77.0")
```
To circle back to this recipe.
Are you fine with bumping the boost version?

Reply:
Originally, this has been one of the main reasons for the new release.
But since there seems to be other ways to override this requirement (`--require-override`) without having to manually change the conanfile, I should be fine with it.

Reply:
And IF there is a binary incompatibility, somewhere you can always force a rebuild of all dependencies using `--build "*"`.